cicerone/app/src/main/java/com/teampower/cicerone/viewmodels/CategoryViewModel.kt

```diff
@@ -47,11 +47,11 @@ class CategoryViewModel(application: Application) : AndroidViewModel(application
         repository.insert(cat)
     }
 
-    fun updateCategoryPoints(catName: String, points: Double) = viewModelScope.launch(Dispatchers.IO) {
-        repository.updateCategoryPoints(catName, points)
+    fun like(catName: String) = viewModelScope.launch(Dispatchers.IO) {
```

Comment:
I like cats too, so catName is a good variable name 😆

Reply:
🐈